modules/theme/src/theme-core/definitions/modules/avatar.less

```diff
@@ -326,7 +332,8 @@
         }
 
         &.massive {
-            height: @massiveWidth;
+            height: @massiveWidth !important;
+            width: @massiveWidth !important;
```

Comment:
What if we move the `!important` to the variable it self so we don't need to duplicate? WDYT?

Reply:
`@massiveWidth` is used in other places as well and might not require `!important` in those places. IMO, would be better to have it enforced here.